code/game/objects/explosion.dm

```diff
@@ -37,7 +37,10 @@
 
 		var/start = world.timeofday
 		epicenter = get_turf(epicenter)
-		if(!epicenter) return
+		if(!epicenter)
+			return
+
+		score["explosions"]++ //For the scoreboard

```

Comment:
This is intended for the Weird category (does not affect the score). Kinda like a benchmark